ebi_eva_common_pyutils/spring_properties.py

```diff
@@ -26,90 +28,153 @@ def __init__(self, maven_profile, private_settings_file):
+        }
 
-spring.data.mongodb.host={mongo_host}
-spring.data.mongodb.port=27017
-spring.data.mongodb.database={accession_db}
-spring.data.mongodb.username={mongo_user}
-spring.data.mongodb.password={mongo_pass}
-
-spring.data.mongodb.authentication-database=admin
-mongodb.read-preference=secondaryPreferred
-spring.main.web-application-type=none
-spring.main.allow-bean-definition-overriding=true
-spring.jpa.properties.hibernate.jdbc.lob.non_contextual_creation=true
-
-parameters.chunkSize=1000
-parameters.assemblyAccession={assembly_accession}
-'''
-
-    def _accessioning_properties(self, instance):
+    def _accessioning_properties(self, instance=''):
         """Properties common to accessioning and clustering pipelines."""
         counts_url, counts_username, counts_password = get_count_service_creds_for_profile(
             self.maven_profile, self.private_settings_file)
-        return f'''
-accessioning.instanceId=instance-{instance}
-accessioning.submitted.categoryId=ss
-accessioning.clustered.categoryId=rs
 
-accessioning.monotonic.ss.blockSize=100000
-accessioning.monotonic.ss.blockStartValue=5000000000
-accessioning.monotonic.ss.nextBlockInterval=1000000000
-accessioning.monotonic.rs.blockSize=100000
-accessioning.monotonic.rs.blockStartValue=3000000000
-accessioning.monotonic.rs.nextBlockInterval=1000000000
+        return {
+            'accessioning.instanceId': self._format_str('instance-{0}', instance),
+            'accessioning.submitted.categoryId': 'ss',
+            'accessioning.clustered.categoryId': 'rs',
+            'accessioning.monotonic.ss.blockSize': 100000,
+            'accessioning.monotonic.ss.blockStartValue': 5000000000,
+            'accessioning.monotonic.ss.nextBlockInterval': 1000000000,
+            'accessioning.monotonic.rs.blockSize': 100000,
+            'accessioning.monotonic.rs.blockStartValue': 3000000000,
+            'accessioning.monotonic.rs.nextBlockInterval': 1000000000,
+            'eva.count-stats.url': counts_url,
+            'eva.count-stats.username': counts_username,
+            'eva.count-stats.password': counts_password
+        }
 
-eva.count-stats.url={counts_url}
-eva.count-stats.username={counts_username}
-eva.count-stats.password={counts_password}
-'''
+    def get_accessioning_properties(self, instance='', target_assembly='',  fasta='', assembly_report='',
+                                    project_accession='', aggregation='BASIC', taxonomy_accession='',
+                                    vcf_file='', output_vcf=''):
+        """Properties for accessioning pipeline."""
+        return self._format(
+            self._common_properties(target_assembly), self._accessioning_properties(instance),
+            {
+                'spring.batch.job.names': 'CREATE_SUBSNP_ACCESSION_JOB',
+                'parameters.assemblyReportUrl': self._format_str('file:{0}', assembly_report),
+                'parameters.chunkSize': 100,
```

Comment:
`chunkSize` is in the common properties

(Unrelated, but I can't remember, why do we set it to 100 for accessioning and 1000 for clustering?)

Reply:
Fixed this.